config/environments/production.rb

```diff
@@ -33,8 +33,8 @@
   # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
 
   # Log to STDOUT with the current request id as a default log tag.
-  config.log_tags = [ :request_id ]
-  config.logger   = ActiveSupport::TaggedLogging.logger(STDOUT)
+  config.log_tags = [:request_id]
+  config.logger = ActiveSupport::TaggedLogging.logger($stdout)
```

Comment:
was there an explanation for this change? seems like these are different things, and i wonder why one vs the other

Reply:
```
 config/environments/production.rb:37:55: Style/GlobalStdStream: Use `$stdout` instead of `STDOUT`.
```